### DISCUSSION TITLE
useFetch status code

### DISCUSSION THREAD

**Discussion by mihail727** [nuxt/nuxt#17228] [Questions] [answered]
How to get status code from server response?
i'm trying write something like this:
```ts
const response = await useFetch(url)
if(resonse.error.status == '401') {
    // do something
}
```

**Comment by mihail727** [comment, marked as answer]
ok, I'm sorry, I already found the answer myself))))
`response.error.value.response.status`

**Reply by ytyng** [reply]
Thank you.

**Comment by ilyaDegtyarenko** [comment]
How to get response status from data? (not error)

**Reply by mihail727** [reply]
Check this method - maybe it help you
https://github.com/nuxt/framework/discussions/4051
